src/tribler-core/tribler_core/modules/bandwidth_accounting/database.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from ipv8.database import Database
+
+from tribler_core.modules.bandwidth_accounting.transaction import BandwidthTransaction
+
+DATABASE_DIRECTORY = "sqlite"
+
+
+class BandwidthDatabase(Database):
```

Comment:
Why don't use Pony here? This is in Tribler, so we get that dependency for free.
Also, don't forget to add the indexes from the beginning. They really help.

Reply:
Overkill atm?

Reply:
Actually, Pony makes code much more readable and easy to modify. We can now even afford a luxury of asking @kozlovsky himiself to help with this :wind_chime:

Reply:
I'm currently rewriting the database logic to use Pony instead, thanks for the advice! 👍